CI/scenarios/post_action_etcd_example_py.py

```diff
@@ -6,7 +6,7 @@
 def run(cmd):
     try:
         output = subprocess.Popen(
-            cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+            cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
```

Comment:
Is this comma supposed to be here? Random commas are at the end of lines in a couple of your files. Haven't seen it cause issues though.

Reply:
Fixed it, thanks.